app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -175,42 +183,47 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>() {
     }
 
     private fun animateRewards(expand: Boolean) {
-        this.showRewardsFragment.removeAllUpdateListeners()
-        this.showRewardsFragment.addUpdateListener { valueAnim ->
-            val initialRadius = resources.getDimensionPixelSize(R.dimen.fab_radius).toFloat()
-            val radius = initialRadius * if (expand) 1 - valueAnim.animatedFraction else valueAnim.animatedFraction
-            this.rewards_container.radius = radius
-        }
+        val showTarget = if (!expand) native_back_this_project_button else pledge_container
```

Comment:
Here I would reserve names starting with verbs for actions. In this case, `showTarget` can makes us believe that it's a function instead of a target of type T. Maybe `targetToShow` is more appropriated? 🤔